test/functional/apps/discover/group6/_sidebar_field_stats.ts

```diff
@@ -298,6 +341,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         expect(await testSubjects.getVisibleText('dscFieldStats-statsFooter')).to.contain(
           '1 sample value'
         );
+
+        await PageObjects.unifiedFieldList.clickFieldListMinusFilter('enabled', 'true');
+        await testSubjects.click('TextBasedLangEditor-expand');
+        const editorValue = await monacoEditor.getCodeEditorValue();
+        expect(editorValue).to.eql(`row enabled = true\n| where \`enabled\`!=true`);
```

Comment:
Would be great to not mix field stats tests and the new filters for ES|QL. Can we extract it into a separate test suit (example: `_esql_where_filters`)? 

The new `filtering by clicking on the table` could be also moved out to a such new file.

Reply:
Julia I am so sorry but I dont agree on this. Filtering is part of the stats so I dont see any reason to create a new file for this. We are going to add more effort to the CI and increase the time without any reason. I am trying to create new files only if there is a good reason and this is not I feel. :/

Reply:
Let's start like that and if we have problems in the future let's re-evaluate